docs/API_SPECS_SIMULATIONS.md

```diff
@@ -458,7 +456,49 @@ Content-Type: application/json; charset=utf-8
 }
 ```
 
-## Get details of the simulation, including device models and statistics
+## Creating a simulation with custom rate limits (aka throttling limits)
+
+The following example shows how to override the rate limits for simulation. Service controls
+the outbound requests to IoT Hub in order to avoid getting throttled. These can be changed 
+according to the targeted IoT Hub.
```

Comment:
I think the paragraph could be rephrased a bit, I'm not sure about the meaning of the 2nd sentence